.github/workflows/ci.yml

```diff
@@ -75,32 +75,22 @@ jobs:
           distribution: 'temurin'
           java-version: ${{ matrix.java }}
           check-latest: true
-      - name: Build Quarkus CLI
-        run: |
-          git clone https://github.com/quarkusio/quarkus.git
-          cd quarkus
-          git checkout 2.2
-          mvn versions:set -DnewVersion=2.2.5.Final -DgenerateBackupPoms=false -pl .,build-parent,devtools/cli
-          cd devtools/cli
-          mvn -B -s ../../../.github/mvn-settings.xml clean install -Dquickly
       - name: Install Quarkus CLI
         run: |
-          cat <<EOF > ./quarkus-dev-cli
-          #!/bin/bash
-          java -jar $PWD/quarkus/devtools/cli/target/quarkus-cli-2.2.5.Final-runner.jar "\$@"
-          EOF
-          chmod +x ./quarkus-dev-cli
-          mkdir -p ~/.quarkus/
-          cp .github/quarkus-cli-config.yaml ~/.quarkus/config.yaml
+          curl -Ls https://sh.jbang.dev | bash -s - app setup
```

Comment:
I have doubts about this change.
The idea of  our GH workflows is to check the latest version of everything, including, I presume, the CLI. And after this change, we will no be checking the latest unpublished CLI binaries.

Reply:
This is for 2.2 branch, we should focus on latest released here. This is not main.

BTW current GH action was using `quarkus-cli-2.2.5.Final-runner.jar`, so we are really moving forward :)